app/Models/Costumer.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Costumer extends Model
```

Comment:
jgn lupa nama filenya juga diganti jadi Customer.php
```suggestion
class Customer extends Model
```